docs/modules/ROOT/partials/setup_terraform.adoc

```diff
@@ -22,6 +22,8 @@ alias terraform='docker run -it --rm \
   --ulimit memlock=-1 \
   ${tf_image}:${tf_tag} terraform'
 
+export GITLAB_REPOSITORY_URL=$(curl -sH "Authorization: Bearer ${COMMODORE_API_TOKEN}" ${COMMODORE_API_URL}clusters/${CLUSTER_ID} | jq -r '.gitRepo.url' | sed 's|ssh://||' | sed 's|/|:|')
+export GITLAB_CATALOG_PROJECT_ID=$(curl -sH "Authorization: Bearer ${GITLAB_TOKEN}" "https://git.vshn.net/api/v4/projects?simple=true&search=${CLUSTER_ID}" | jq -r ".[] | select(.ssh_url_to_repo == \"${GITLAB_REPOSITORY_URL}\") | .id")
```

Comment:
The original version doesn't work when the cluster catalog repository name doesn't match the cluster's ID in Lieutenant. 

```suggestion
export GITLAB_REPOSITORY_NAME=${GITLAB_REPOSITORY_URL##*/}
export GITLAB_CATALOG_PROJECT_ID=$(curl -sH "Authorization: Bearer ${GITLAB_TOKEN}" "https://git.vshn.net/api/v4/projects?simple=true&search=${GITLAB_REPOSITORY_NAME/.git}&per_page=100" | jq -r ".[] | select(.ssh_url_to_repo == \"${GITLAB_REPOSITORY_URL}\") | .id")
```